modules/nf-httpfs/src/main/nextflow/file/http/LatchPath.groovy

```diff
@@ -340,6 +344,7 @@ class LatchPath extends XPath {
 
                 HttpRequest req =  HttpRequest.newBuilder()
                     .uri(URI.create(chunkUrl))
+                    .timeout(Duration.ofSeconds(10))
```

Comment:
maybe we should bump this since we are actually uploading data? 
having this be a higher value like 30/60 seconds is probably fine because it works in most cases and we are just trying to solve the case where it hangs indefinitely

Reply:
good point, i did 90 like we have in sdk